orthoinference/src/main/java/org/reactome/orthoinference/InferEWAS.java

```diff
@@ -0,0 +1,354 @@
+package org.reactome.orthoinference;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaClass;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.InvalidAttributeValueException;
+import org.gk.schema.SchemaClass;
+
+public class InferEWAS {
```

Comment:
Naming convention: classes are usually named as _nouns_. So this would be "EWASInferrer".